packages/flutter_tools/lib/src/dart/language_version.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:file/file.dart';
+import 'package:package_config/package_config.dart';
+
+final RegExp _languageVersion = RegExp('/\/\\s*@dart');
+final RegExp _declarationEnd = RegExp('(import)|(library)|(part)');
+const String _blockCommentStart = '/*';
+const String _blockCommentEnd = '*/';
+
+/// Attempts to read the language version of a dart [file].
```

Comment:
The doc should be clear that the returned String _is_ a language version override comment.